src/lib/viewers/doc/DocBaseViewer.js

```diff
@@ -99,6 +99,8 @@ class DocBaseViewer extends BaseViewer {
         // Call super() to set up common layout
         super.setup();
 
+        this.thumbnailsSidebarEl = this.options.container.querySelector('.bp-thumbnails-container');
```

Comment:
Super minor, but it seems like this should happen last. The core preview experience is the most important thing going on in this method.